openedx/openedx-platform
# feat: add url to usage location

## Description

This PR updates the API endpoints for assets and videos that fetch the usage locations to include a link to the block in a unit. The link adds a hash to the specific block which also the page to scroll the block into view once the page is fully loaded. This change impacts Authors.

## Supporting information

JIRA Ticket: [TNL-11158](https://2u-internal.atlassian.net/browse/TNL-11158)

> This list is found on bottom left hand side of info modal. Users can currently see if a given asset is in the course, and how many times it is in the course. What is considered a “game changer” is to make that list clickable so that they can know where in the course the asset is. 

## Testing instructions

1. Open a course and navigate to the Files page (MFE version)
2. Open the info modal for an asset that you know is used in the course
3. Info modal should show all the locations that the asset is used and on hover show a valid link
4. Click on one of the links
5. Unit page should open in a new tab
6. One the xblocks finish loading, the xblock that uses the asset should scroll into view.

## Deadline

None

## Other information

This PR should use `course-authoring` PR #[717](https://github.com/openedx/frontend-app-course-authoring/pull/717) for testing
